webpack.config.js

```diff
@@ -9,7 +9,9 @@ module.exports = {
 	entry: "./src/index.js",
 	output: {
 		path: './build',
-		filename: name + '.js'
+		filename: name + '.js',
+		library: name,
+		libraryTarget: 'commonjs',
```

Comment:
This is what makes it so that it calls `require('angular')` instead of `window.angular`